server/server_endpoints.go

```diff
@@ -57,7 +56,12 @@ func (s *Server) createOpenAPIInterfaceImpl() (impl api.StrictServerInterface, e
 		return nil, fmt.Errorf("no refresh API implementation found %w", err)
 	}
 
-	return api.NewOpenAPIInterfaceImpl(bControl, s, refresher), nil
+	cacheControl, err := resolver.GetFromChainWithType[api.CacheControl](s.queryResolver)
```

Comment:
This won't clear the bootstrap cache. I don't think that's really necessary, but just wanted to make note of it in case you want to document/implement it.